docs/source/host-collect-analyze/overview.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Overview
+description: Host Collectors and Analyzers in Support Bundles
+---
+
+In the event you need to collect and analyze information that isn't available when using in-cluster collectors, you can use Host Collectors to gather information about the environment such as CPU, memory, available block devices, etc. This can be useful when you need to collect information at the host level.
```

Comment:
The last sentence is redundant. Can you more clarity on when this would be useful?